src/langsmith/assistants.mdx

```diff
@@ -3,36 +3,179 @@ title: Assistants
+  - Customize model selection, system prompts, or tool availability per user.
+  - Store user preferences and apply them automatically to each interaction.
+  - Enable users to choose between different AI personalities or expertise levels.
+
+- **Customer or organization-specific configurations**
+  - Maintain separate configurations for different customers or organizations.
+  - Customize behavior for each client without deploying separate infrastructure.
+  - Isolate configuration changes to specific customers.
+
+```mermaid
+graph TD
+    A["Graph: agent<br/>(deployed)"]
+    A --> B["Customer A Assistant<br/>━━━━━━━━━━━━━<br/>Model: GPT-4<br/>Tone: Legal<br/>Tools: Custom"]
+    A --> C["Customer B Assistant<br/>━━━━━━━━━━━━━<br/>Model: Claude<br/>Tone: Casual<br/>Tools: Standard"]
+    A --> D["Customer C Assistant<br/>━━━━━━━━━━━━━<br/>Model: GPT-3.5<br/>Tone: Formal<br/>Tools: Limited"]
+
+    style A fill:#4A90E2,stroke:#2E5C8A,stroke-width:3px,color:#fff
+    style B fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+    style C fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+    style D fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+```
+
+- **Environment-specific configurations**
+  - Use different models or settings for development, staging, and production.
+  - Test configuration changes in staging before promoting to production.
+  - Reduce costs in non-production environments with smaller models.
+
+- **A/B testing and experimentation**
+  - Compare different prompts, models, or parameter settings.
+  - Roll out configuration changes gradually to a subset of users.
+  - Measure performance differences between configuration variants.
+
+- **Specialized task variants**
+  - Create domain-specific versions of a general-purpose agent.
+  - Optimize configurations for different languages, regions, or industries.
+  - Maintain consistent graph logic while varying the execution details.
+
+```mermaid
+graph TD
+    A["Graph: writing-agent<br/>(deployed)"]
+    A --> B["Blog Assistant<br/>━━━━━━━━━━━━━<br/>Model: GPT-4<br/>Tone: Formal<br/>Style: Long-form<br/>Tools: SEO optimization"]
+    A --> C["Tweet Assistant<br/>━━━━━━━━━━━━━<br/>Model: GPT-4-mini<br/>Tone: Casual<br/>Style: 280-char limit<br/>Tools: Hashtag suggestions"]
+    A --> D["Email Assistant<br/>━━━━━━━━━━━━━<br/>Model: GPT-4<br/>Tone: Professional<br/>Style: Medium length<br/>Tools: Templates"]
+
+    style A fill:#4A90E2,stroke:#2E5C8A,stroke-width:3px,color:#fff
+    style B fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+    style C fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+    style D fill:#E8F4F8,stroke:#4A90E2,stroke-width:2px
+```
+
+## How assistants work with deployments
+
+When you deploy a graph with LangSmith Deployment, [Agent Server](/langsmith/agent-server) automatically creates a **default assistant** tied to that graph's default configuration. You can then create additional assistants for the same graph, each with its own configuration.
```

Comment:
We should mention this here (somehow): Each graph defined in `langgraph.json` gets a default assistant.
```json
{
    "graphs": {
        "graph_id_1": "path_to_graph_id_1",  // <-- default assistant is created for graph_id_1
        "graph_id_2": "path_to_graph_id_2"  // <-- default assistant is created for graph_id_2
    }
}
```

In other words, there can be multiple "default" assistants (one for each graph defined in `langgraph.json`).